Modules/Microsoft365DSC/DSCResources/MSFT_EXOAtpPolicyForO365/readme.md

```diff
@@ -36,6 +36,13 @@ AllowClickThrough
   users to click through to the original blocked URL in
   Office 365 ProPlus. The default value is $true
 
+AllowSafeDocsOpen
```

Comment:
Also, for future PR, the fields' description is also automatically pulled from the .schema.mof description, so no need to repeat them in the readme.md. That will save you time in the future ;)